apps/kahoot-it-clone/src/app/quiz/quiz-bank/quiz-bank.component.html

```diff
@@ -0,0 +1,41 @@
+<section>
+  <div cdkDropList class="quizList" (cdkDropListDropped)="drop($event)">
+    <div
+      class="quizBox quizBank"
+      *ngFor="let question of questions.controls; let i = index"
+      cdkDrag
+      (click)="selectQuestion(i)"
+    >
+      <mat-label class="questionTitle"> Question {{ i + 1 }}</mat-label>
+      <div class="question">
+        {{ question.value.question }}
+      </div>
+      <div class="image">
+        {{ question.value.image }}
+      </div>
+      <div class="answers">
+        {{ question.value.answers }}
+      </div>
+      <div class="duplicateGrid">
+        <button mat-raised-button class="button delete">Duplicate</button>
+      </div>
+      <div class="deleteGrid">
+        <button mat-raised-button class="button delete">Delete</button>
+      </div>
+    </div>
+  </div>
+</section>
+
```

Comment:
Is this supposed to be the preview of the question component? If so this might be easier to just throw that the question component in here and scale it down if they are effectively the same thing.